test/pkg/environment/common/setup.go

```diff
@@ -135,7 +135,6 @@ func (env *Environment) CleanupObjects(cleanableObjects ...client.Object) {
 					defer GinkgoRecover()
 					g.Expect(env.ExpectTestingFinalizerRemoved(&metaList.Items[i])).To(Succeed())
 					g.Expect(client.IgnoreNotFound(env.Client.Delete(env, &metaList.Items[i],
```

Comment:
Can we use https://github.com/awslabs/operatorpkg/blob/main/test/expectations/expectations.go#L156

Reply:
Just tried this. We need some way to only delete objects with labels that we care about, which we use test.DiscoveryLabel right now to only delete nodes that were created in the test. Happy to include in a followup when that capability is added in

Reply:
Tried removing nodes from the list, the cleanup also cleans up all namespaces for resources, which we want to keep around across tests